config/config_test.go

```diff
@@ -78,6 +78,54 @@ func TestOnlyEnvConfig(t *testing.T) {
 	os.Setenv("DD_API_KEY", "")
 }
 
+func TestOnlyEnvConfigArgsScrubbingEnabled(t *testing.T) {
+	os.Setenv("DD_SCRUB_ARGS", "true")
```

Comment:
Can we verify that this defaults to true correctly as well without setting this explicitly to `true`?

Reply:
Yep, it sounds a good idea. I've changed it